src/main/java/com/google/api/codegen/py/PythonImportHandler.java

```diff
@@ -238,11 +218,13 @@ public String addImportLocal(String moduleName, String attributeName) {
     return all;
   }
 
-  public String fileToModule(ProtoFile file) {
-    if (fileImports.containsKey(file)) {
-      return fileImports.get(file);
-    } else {
-      return "";
+  /** Returns the name for a module for a module and attribute stored in the input handler. */
+  public String getShortName(String module, String attribute) {
+    for (PythonImport pyImport : stringImports.inverse().keySet()) {
+      if (pyImport.attributeName().equals(attribute) && pyImport.moduleName().equals(module)) {
+        return pyImport.shortName();
```

Comment:
This can be changed to be a constant time lookup if the local name is removed from PythonImport since it is essentially stored as the shortname in the stringImports map.

Thoughts?